client/src/components/Q&A/Q&Alist.jsx

```diff
@@ -1,30 +1,28 @@
+/* eslint-disable react/prop-types */
 import React from 'react';
-import App from '../App.jsx';
-import axios from 'axios';
-import Question from './Question.jsx';
+import QuestionListEntry from './QuestionListEntry.jsx';
 
 const QuestionList = (props) => {
   console.log(props);
-    return (
-      <div className="container"><p class="text-start">
-        Questions and Answers
-      </p>
-      <div class="input-group mb-3">
-        <input type="text" class="form-control" placeholder="HAVE A QUESTION? SEARCH FOR ANSWERS..." aria-label="Recipient's username" aria-describedby="button-addon2"></input>
-          <button class="btn btn-outline-secondary" type="button" id="button-addon2">Button</button>
-      </div>
-      <div className="row">
-          {props.data.QuestionListdata.QuestionListForId5.results.map((question) => {
-            return <Question data={question} />;
-          })}
-      </div>
-      </div>
-    )
-}
+  return (
+    <div className="container"><p className="text-start fs-5 fw-light">
+      Questions and Answers
+    </p>
+    <div className="input-group mb-3">
+      <input type="text" className="form-control" placeholder="HAVE A QUESTION? SEARCH FOR ANSWERS..." aria-label="Recipient's username" aria-describedby="button-addon2"></input>
+      <button className="btn btn-outline-secondary" type="button" id="button-addon2">Button</button>
+    </div>
+    <div className="row">
+      {props.data.results.map((question, index) => {
```

Comment:
What happens if props.data.results is undefined or an empty array?

Reply:
That's actually the same question I have still. There is nothing in the Business Reqs about what to do when there are no questions for a given product, or when there are no answers for a given question. Guessing I'll have to put a placeholder image or something.